### PR TITLE
vmspawn: fix `sd_bus_message_append()` arguments and add missing error check

### PR DESCRIPTION
**Pull request by aafeijoo-suse**

### CODE REVIEW

File: src/vmspawn/vmspawn-scope.c [hunk 1]
```diff
@@ -60,6 +60,8 @@ int start_transient_scope(sd_bus *bus, const char *machine_name, bool allow_pidf
                                   "Description", "s",  description,
```
Comment (YHNdnzj) [inline src/vmspawn/vmspawn-scope.c:60]: Hmm, `(sa)` should be replaced by `(sv)` above too...

**Review by YHNdnzj — COMMENTED** [review]
Not your fault, but the code contains more problems :/

**Comment by YHNdnzj** [comment]
So this stuff ever worked _because the error check is missing_, wild...

File: src/vmspawn/vmspawn-scope.c (hunk 1 above)
Comment (aafeijoo-suse) [reply src/vmspawn/vmspawn-scope.c:60]: ah, I missed that...

**Review by YHNdnzj — APPROVED** [review]

**Pull request merged**